docs/introduction/Car-registry-tutorial.rst

```diff
@@ -309,29 +305,28 @@ AbstractRegularTransaction
 
 *CarDeclarationTransaction* extends previously declared *AbstractRegularTransaction* in the following way: ``public final class CarDeclarationTransaction extends AbstractRegularTransaction``
-newBoxes() -- a new box with a newly created car shall be added as well. Thus that function shall be overridden as well for adding new CarBox additional to regular boxes.  
+newBoxes() -- a new box with a newly-created car shall be added as well. Thus, that function shall be overridden as well for adding new CarBox additional to regular boxes.  
 
 SellCarTransaction 
 ******************
 
-*SellCarTransaction* extends previously declared AbstractRegularTransaction in next way: ``public final class SellCarTransaction extends AbstractRegularTransaction``
-Similar to *CarDeclarationTransaction, newBoxes()* function shall also return a new specific box. In our case that new box is *CarSellOrderBox*. Also due we have specific box to open (CarBox), we also need to add unlocker for CarBox, so unlocker for that CarBox had been added in ``public List<BoxUnlocker<Proposition>> unlockers()``
+*SellCarTransaction* extends previously declared AbstractRegularTransaction in next (following?) way: ``public final class SellCarTransaction extends AbstractRegularTransaction``
+Similar to *CarDeclarationTransaction, newBoxes()* function shall also return a new specific box. In our case that new box is *CarSellOrderBox*. Also, since we have a specific box to open (CarBox), we also need to add unlocker for CarBox, so unlocker for that CarBox had been added in ``public List<BoxUnlocker<Proposition>> unlockers()``
 
 BuyCarTransaction
 *****************
 
-Few comments about implementation: 
-During the creation of unlockers in function *unlockers()*, we need to also create a specific unlocker for opening a car sell order. Another *newBoxes()* function has a bit specific implementation. That function forces to create a new RegularBox as payment for a car in case the car has been sold. Anyway, a new Car box also shall be created according to information in ``carBuyOrderInfo``. 
+A few comments about implementation: 
+During the creation of unlockers in function *unlockers()*, we need to also create a specific unlocker for opening a car sell order. Another *newBoxes()* function has a bit-specific implementation. That function forces creation of a new RegularBox as payment for a car in case the car has been sold. Anyway, a new Car box also shall be created according to information in ``carBuyOrderInfo``. 
 
 Extend API: 
 ***********
 
-* Create a new class CarApi which extends ApplicationApiGroup class, add that new class to Route by it in SimpleAppModule, like described in Custom API manual. In our case it is done in ``CarRegistryAppModule`` by 
+* Create a new class CarApi which extends ApplicationApiGroup class, add that new class to Route by it in SimpleAppModule, as described in the Custom API manual. In our case it is done in ``CarRegistryAppModule`` by 
```

Comment:
@zennermac 

* Create a new class CarAPI which extends ApplicationAPIGroup class. Add this new class to route it in SimpleAppModule, as described in the Custom API manual. In our case it is done in ``CarRegistryAppModule`` by